Sources/SwiftGenKit/Stencil/JSONContext.swift

```diff
@@ -0,0 +1,39 @@
+//
+// SwiftGen
+// Copyright © 2019 SwiftGen
+// MIT Licence
+//
+
+import Foundation
+
+//
+// See the documentation file for a full description of this context's structure:
+// Documentation/SwiftGenKit Contexts/Yaml.md
+//
+
+extension JSON.Parser {
+  public func stencilContext() -> [String: Any] {
```

Comment:
_Note to self, just realised that while reviewing here: why don't we call those files `JSONParser+Context.swift` instead of `JSONContext.swift`? Maybe some cleanup to do in a separate PRs one day…_

Reply:
Adressed in #729